src/main/java/frc/robot/commands/ConveyorCommands/ConveyInwardCommand.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands.ConveyorCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ConveyorSubsystem;
+
+public class ConveyInwardCommand extends Command{
+    ConveyorSubsystem conveyor = new ConveyorSubsystem();
```

Comment:
You don't want to create a `new ConveyorSubsystem()` here since it's being assigned in the constructor.

Reply:
sweeping up the trash 🧹